assets/html/error-pages/service-error-std.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <title>
+            Sorry, there's a problem with the service - Office for National Statistics
+        </title>
+
+        <meta charset="utf-8">
+        <meta
+            content="width=device-width,initial-scale=1.0,user-scalable=1"
+            name="viewport"
+        >
+        <meta
+            name="format-detection"
+            content="telephone=no"
+        >
+        <meta
+            name="theme-color"
+            content="#58595B"
+        >
+        <meta
+            name="apple-mobile-web-app-status-bar-style"
+            content="#58595B"
+        >
+
+        <link
+            rel="stylesheet"
+            href="https://cdn.ons.gov.uk/dp-design-system/a19df14/css/main.css"
+        >
+        <link
+            rel="shortcut icon"
+            href="https://cdn.ons.gov.uk/assets/images/favicon.ico"
+            type="image/x-icon"
+        >
+    </head>
+
+    <body>
+        <div class="ons-page">
+            <div class="ons-page__content">
+                <header class="ons-u-bb">
+                    <div class="wrapper">
+                        <div class="header col-wrap">
+                            <div class="col col--lg-one-third col--md-one-third">
+                                <a
+                                    id="logo-link"
+                                    href="/"
+                                >
+                                    <img
+                                        class="logo"
+                                        src="https://cdn.ons.gov.uk/assets/images/ons-logo/v2/ons-logo.svg"
+                                        alt="Office for National Statistics"
+                                    >
+                                </a>
+                            </div>
+                        </div>
+                    </div>
+                </header>
+
+                <main>
+                    <div class="ons-page__container ons-container">
+                        <div class="ons-grid ons-u-ml-no">
+                            <div class="ons-grid__col ons-u-pl-no">
+                                <h1 class="ons-u-mt-xl ons-u-fw-b">
+                                    Sorry, there's a problem with the service
+                                </h1>
+                                <div class="ons-page__main ons-u-mt-s">
+                                    <p>Try again later.</p>
+                                    <p>
+                                        Our scheduled publications can also be found on our
+                                        <a href="https://backup.ons.gov.uk/">backup site</a>.
+                                    </p>
+                                    <p>If you have any questions,
+                                        <a
+                                            href="https://www.ons.gov.uk/aboutus/contactus/generalandstatisticalenquiries">contact
```

Comment:
bit inconsistent to have contact us as a web link here,
and then in the other file (below), change it from a web link (line 75 in that file, too) to an email link

Reply:
The STD file is so its easy for us to flip it back 👍 
The new page is going on the assumption the site is not accessible so not really worth giving them a link to the site they can't access